scripts/eurostat/health_determinants/common/euro_stat.py

```diff
@@ -0,0 +1,343 @@
+                                  "input_files")
+flags.DEFINE_string("input_path", default_input_path, "Import Data File's List")
+
+_TMCF_TEMPLATE = ("Node: E:eurostat_population_{import_name}->E0\n"
+                  "typeOf: dcs:StatVarObservation\n"
+                  "variableMeasured: C:eurostat_population_{import_name}->SV\n"
+                  "measurementMethod: C:eurostat_population_{import_name}->"
+                  "Measurement_Method\n"
+                  "observationAbout: C:eurostat_population_{import_name}->geo\n"
+                  "observationDate: C:eurostat_population_{import_name}->time\n"
+                  "scalingFactor: 100\n"
+                  "value: C:eurostat_population_{import_name}->observation\n")
+
+
+class EuroStat:
+    """
+    EuroStat is a base class which provides common implementation for generating
+    CSV, MCF and tMCF files.
+    """
+    # Below variables will be initialized by sub-class (import specific)
+    _import_name = ""
+    _mcf_template = ""
+    _sv_value_to_property_mapping = {}
+    _sv_properties_template = {}
+    _sv_properties = {}
+
+    def __init__(self,
+                 input_files: list,
+                 csv_file_path: str = None,
+                 mcf_file_path: str = None,
+                 tmcf_file_path: str = None) -> None:
+        self._input_files = input_files
+        self._cleaned_csv_file_path = csv_file_path
+        self._mcf_file_path = mcf_file_path
+        self._tmcf_file_path = tmcf_file_path
+        self._df = pd.DataFrame()
+
+    # pylint: disable=pointless-statement
+    # pylint: disable=no-self-use
+    # pylint: disable=unused-argument
+
+    def set_cleansed_csv_file_path(self, cleansed_csv_file_path: str) -> None:
+        self._cleaned_csv_file_path = cleansed_csv_file_path
+
+    def set_mcf_file_path(self, mcf_file_path: str) -> None:
+        self._mcf_file_path = mcf_file_path
+
+    def set_tmcf_file_path(self, tmcf_file_path: str) -> None:
+        self._tmcf_file_path = tmcf_file_path
+
+    def _property_correction(self):
+        None
+
+    def _sv_name_correction(self, sv_name: str) -> str:
+        None
+
+    def _rename_frequency_column(self, df: pd.DataFrame) -> pd.DataFrame:
+        return df
+
+    # pylint: enable=pointless-statement
+    # pylint: enable=unused-argument
+    def _parse_file(self, file_name: str, df: pd.DataFrame,
+                    import_name: dict) -> pd.DataFrame:
+        """
+        Parsing the input file, loading to dataframe and cleaning it.
+
+        Args:
+            file_name (str): name of the input file.
+            df (pd.DataFrame):
+            import_name (dictonary): dictonary of all eurostat imports.
+
+        Returns:
+            df (pd.DataFrame):
+        """
+        original_df_columns = df.columns
+        df = split_column(df, df.columns.values.tolist()[0])
+
+        df.rename(columns={
+            'geo\time': 'geo',
```

Comment:
`\t` is interpreted as a tab in a string. You can use raw strings to use a slash: `r'geo\time'`. Then you may not need `'geo\\time'` below.